src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -2,140 +2,17 @@
   <traction-modal ref="well-modal" :static="isStatic" size="lg">
     <template #modal-title> Add Pool to Well: {{ position }} </template>
 
-    <traction-form>
-      <traction-form-group id="movie-time-group" label="Movie time:" label-for="movie-time">
-        <traction-select
-          id="movie-time"
-          ref="movieTime"
-          v-model="well.movie_time"
-          data-attribute="movie-time"
-          :options="movieTimeOptions"
-        >
-        </traction-select>
+    <fieldset>
```

Comment:
@stevieing I think this should still be <traction-form> but <traction-form-group >should be <fieldset>?

Reply:
> @stevieing I think this should still be traction-form but traction-form-group should be fieldset?

We can leave this to Ene as she has a branch in WIP. She will need to merge in develop anyway.